src/zenml/cli/utils.py

```diff
@@ -1021,20 +1025,27 @@ def install_packages(
         )
         uninstall_package("neptune-client")
 
+    pip_command = ["uv", "pip"] if use_uv else ["pip"]
     if upgrade:
-        command = [
-            sys.executable,
-            "-m",
-            "pip",
-            "install",
-            "--upgrade",
-        ] + packages
+        command = (
+            [
+                sys.executable,
+                "-m",
+            ]
+            + pip_command
+            + [
+                "install",
+                "--upgrade",
```

Comment:
Feels like we need `--system` for `uv`

Reply:
Just in case, I get right, that if venv is set, but `--system` is passed we still install into venv?

Reply:
No, it might be good as is, but then I have another request: can you add a catcher for the cases when Venv is not there and print properly?

Reply:
Ok yes that's a good point. I'm pretty sure that `uv` will spit out its own error, but I can catch it and make it a nice error.

Reply:
Done